Commons/bukkit/src/main/java/tc/oc/commons/bukkit/report/ReportCommands.java

```diff
@@ -184,7 +189,7 @@ public void reports(final CommandContext args, final CommandSender sender) throw
                         audience.sendMessage(new HeaderComponent(title));
                         for(Report report : reportResult.documents()) {
                             if(report.reported() != null) {
```

Comment:
You should  check if the player is offline here instead of in ReportFormatter. That would avoid calling `audience.sendmessage()` for a bunch of empty collections.